api/api_ethereum.go

```diff
@@ -792,15 +793,19 @@ func newEthRPCTransaction(tx *types.Transaction, blockHash common.Hash, blockNum
 		result.GasTipCap = (*hexutil.Big)(tx.GasTipCap())
 		// TODO-Klaytn: If we change the gas price policy from fixed to dynamic,
 		// we should change the params.ZeroBaseFee(fixed value) to dynamic value.
-		price := math.BigMin(new(big.Int).Add(tx.GasTipCap(), new(big.Int).SetUint64(params.ZeroBaseFee)), tx.GasFeeCap())
-		result.GasPrice = (*hexutil.Big)(price)
+		if b != nil && b.Header().BaseFee != nil {
+			result.GasPrice = (*hexutil.Big)(b.Header().BaseFee)
```

Comment:
I think we need to return the gasPrice of the transaction (what user sent) in GasPrice, and baseFee should be returned in `effectiveGasPrice` filed.

Reply:
Oh sorry, never mind. This is for TxTypeEthereumDynamicFee